### PR TITLE
Fix release workflows to disconnect from pypi 

### PR DESCRIPTION
**Pull request by Treesarj**
# Description:

## Type of change:

- [ ] Bug fix.
- [ ] New feature.
- [ ] Documentation update.

## Checklist for the reviewer:

This checklist should be used as a help for the reviewer.

- [ ] Is the change limited to one issue?
- [ ] Does this PR close the issue?
- [ ] Is the code easy to read and understand, including clearly named variables?
- [ ] Do all new feature have an accompanying new test?
- [ ] Has the documentation been updated as necessary?

### CODE REVIEW

**Comment by Treesarj** [comment]
Maybe we can remove certain steps in the CD_release.yml as it doesn't have access to certain env variables /secrets?

File: .github/workflows/cd_release.yml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+    - name: Build source distribution
+      run: python -m build
+
+    - name: Publish package to PyPI
+      uses: pypa/gh-action-pypi-publish@v1.8.14
+      with:
+        user: __token__
+        password: ${{ secrets.PYPI_TOKEN }}
+        publish_on_pypi: false
```
Comment (CasperWA) [inline .github/workflows/cd_release.yml]: Remove this step altogether.

**Review by CasperWA — CHANGES_REQUESTED** [review]
I thought we were using `SINTEF/ci-cd` here. That workflow has an option to _not_ publish on PyPI using the `publish_on_pypi` input parameter. This one does not.
But here it is as easy as removing that step in the workflow.

Indeed, for this to work in a more optimized way, `SINTEF/ci-cd` callable workflows should be used. In any case - this is fine, as long as the publish to pypi step is removed :)

**Review by CasperWA — APPROVED** [review]
This should be fine :)

**Pull request merged**